From 6c777445050d715512acd2ac0d419992bc224da7 Mon Sep 17 00:00:00 2001
From: Gyorgy Sarvari <skandigraun@gmail.com>
Date: Wed, 26 Nov 2025 13:06:22 +0100
Subject: [PATCH] lower cmake requirement

Kirkstone has CMake 3.22, but the minimum is specified as 3.26.

Lower it, so it can be built on Kirkstone - the cmake files are
fairly straightforward and simple, without very arcane options.

Upstream-Status: Inappropriate [oe-specific]
Signed-off-by: Gyorgy Sarvari <skandigraun@gmail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a0d3fca..22ef298 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,7 +6,7 @@
 # WASI sysroot build requires this previous compiler and must be runnable on
 # the host.
 
-cmake_minimum_required(VERSION 3.26)
+cmake_minimum_required(VERSION 3.22)
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 project(wasi-sdk)
 include(ExternalProject)
